lib/request.js

```diff
@@ -257,15 +257,14 @@ module.exports = {
    */
 
   get fresh() {
-    const method = this.method;
-    const s = this.ctx.status;
+    const { method, ctx: { status, response }, header } = this;
```

Comment:
I guess I would write this as:

```js
const { method, ctx } = this
const { status, response } = ctx
```

or maybe just have one assignment per line:

```js
const {
  method: {
    ctx: {
      status,
      method
    }
  }
} = this
```

not a fan of complex single-line stuff